Sources/Specs/ViewSpec.swift

```diff
@@ -0,0 +1,19 @@
+import SnowManLogic
+import Swiftest
+
+class ViewSpec: Swiftest.Spec {
+  	let spec = describe("view logic") {
+  		var ui: Receivable!
+  		var view: View!
+  		before() {
+  			ui = MockUI()
+  			view = View(ui: ui)
+  		}
+  		
+  		describe("#reveiveGuess") {
+  			it("should return the guess if it is a letter or word") {
```

Comment:
this test description should describe what's happening and where the letter is coming from